src/lammpsio/snapshot.py

```diff
@@ -437,6 +462,106 @@ def has_mass(self):
         """
         return self._mass is not None
 
+    @property
+    def bonds(self):
+        """:class:Bonds: bonds."""
```

Comment:
Do like this:
```suggestion
        """:class:`lammpsio.topology.Bonds`: Bond data."""
```
Carry forward